vitrina/datasets/urls.py

```diff
@@ -196,10 +196,15 @@
         name="dataset-rdf-download",
     ),
     path(
-        "datasets/<int:pk>/structure/import/",
+        "datasets/<int:pk>/version/<int:version_id>/structure/import/",
```

Comment:
```suggestion
        "datasets/<int:pk>/versions/<int:version_id>/structure/import/",
```
As `import` is a action we do, it can also be emitted:
```
POST /datasets/<int:pk>/versions/<int:version_id>/structure
```

Reply:
Removed `import`